aptos-move/e2e-move-tests/src/tests/stake.rs

```diff
@@ -81,33 +80,6 @@ fn test_staking_end_to_end() {
     assert_eq!(stake_pool.inactive, 0);
 }
 
-#[test]
-fn test_staking_mainnet() {
```

Comment:
This is an outdated test that doesn't use the right mainnet configuration. We have a new mainnet flow that's tested by a test in vm-genesis/src/lib.rs now